src/common/fission.ts

```diff
@@ -0,0 +1,80 @@
+import * as DOH from "../components/reference/dns-over-https.js"
+
+
+/**
+ * Fission endpoints.
+ *
+ * `apiPath` Path of the API on the Fission server.
+ * `lobby` Location of the authentication lobby.
+ * `server` Location of the Fission server.
+ * `userDomain` User's domain to use, will be prefixed by username.
+ */
+export type Endpoints = {
+  apiPath: string
+  lobby: string
+  server: string
+  userDomain: string
+}
+
+
+export const PRODUCTION: Endpoints = {
+  apiPath: "/v2/api",
+  lobby: "https://auth.fission.codes",
+  server: "https://runfission.com",
+  userDomain: "fission.name"
+}
+
+
+export const STAGING: Endpoints = {
+  apiPath: "/api/v2",
```

Comment:
```suggestion
  apiPath: "/v2/api",
```
Should this be `/v2/api`?

Reply:
Thanks for spotting that! 👍